projects/packages/forms/src/contact-form/css/file-field.css

```diff
@@ -111,6 +110,7 @@
     background-size: cover;
     border-radius: var(--jetpack--contact-form--border-radius, 0);
     outline: 2px solid  var( --jetpack--contact-form--button-primary--text-color, #FFF );
+	background-color: #333;
 	border-radius: 50%;
```

Comment:
Note mixed spacing tabs/spaces in this file.

Reply:
Good catch!